### PR TITLE
NEXTPY-1587 -- hasChange method in dynamic converter

### PR DESCRIPTION
**Pull request by kdleijer**
hasChange method in dynamic converter should not execute but pass down its check to the child converter

### CODE REVIEW

File: src/dynamic-converter.ts [hunk 1]
```diff
@@ -59,8 +59,16 @@ function delegatingConverter<R, V>(
         options
       );
     },
-    hasChange(currentValue: V, newValue: V): boolean {
-      return currentValue !== newValue;
```
Comment (kdleijer) [inline src/dynamic-converter.ts:63]: This was a bug in the initial PR, we should pass down the hasChange event to the child converter.
Re-enable this line and see the bug in the additional testcase

**Review by HarmDonkers — APPROVED** [review]
Code review OK

Checked for potential security issues
Checked for code optimization
Checked for code duplication
Checked for whether names are descriptive enough
Checked for possible performance improvements
Checked for the presence and quality of tests
Checked for additional code documentation
Checked for backwards compatibility

**Review by randywanga — APPROVED** [review]
This should fix the previous problems. 👍

**Review by HarmDonkers — APPROVED** [review]
Code review OK

Checked for potential security issues
Checked for code optimization
Checked for code duplication
Checked for whether names are descriptive enough
Checked for possible performance improvements
Checked for the presence and quality of tests
Checked for additional code documentation
Checked for backwards compatibility

**Pull request merged**